src/analysis/retail/demonhunter/vengeance/modules/core/MajorDefensives/core.tsx

```diff
@@ -0,0 +1,389 @@
+import styled from '@emotion/styled';
```

Comment:
missed reviewing this but there's a lot of copying here. when possible, if you want to use something from another spec we should get it moved to the `shared` section rather than duplicating the code

Reply:
I'll move the non-changed pieces over to a shared section, I was a bit worried I might have to make some changes due to VDH weirdness and didn't want to affect Brew.